db/seeders/data/pending-applications.js

```diff
@@ -0,0 +1,449 @@
+'use strict';
+const faker = require('faker');
+const moment = require('moment');
+const uuidv4 = require('uuid/v4');
+
+const projectCountries = ['scotland', 'wales', 'england', 'northern-ireland'];
+const orgTypes = [
+    'unregistered-vco',
+    'unincorporated-registered-charity',
+    'charitable-incorporated-organisation',
+    'not-for-profit-company',
+    'school',
+    'college-or-university',
+    'statutory-body',
+    'faith-group'
+];
+
+const toDateParts = dt => {
+    return { day: dt.date(), month: dt.month() + 1, year: dt.year() };
+};
+
+const mockAddress = () => {
+    return {
+        line1: faker.address.streetAddress(),
+        townCity: faker.address.city(),
+        county: faker.address.county(),
+        postcode: 'B15 1TR'
+    };
+};
+
+module.exports = {
+    data: [
+        {
+            id: uuidv4(),
+            userId: 1,
+            formId: "awards-for-all",
+            applicationData: JSON.stringify({
+                projectName: faker.lorem.words(5),
+                bankSortCode: faker.finance.account(6),
+                bankStatement: {
+                    size: 123,
+                    type: "application/pdf",
+                    filename: "example.pdf"
+                },
+                charityNumber: faker.finance.account(7),
+                projectBudget: [
+                    {
+                        cost: faker.random.number({ min: 100, max: 300 }),
+                        item: faker.lorem.words(5)
+                    }
+                ],
+                projectCountry: faker.random.arrayElement(projectCountries),
+                bankAccountName: faker.finance.accountName(),
+                mainContactName: {
+                    lastName: faker.name.lastName(),
+                    firstName: faker.name.firstName()
+                },
+                projectLocation: "east-lothian",
+                projectPostcode: mockAddress().postcode,
+                termsPersonName: `${faker.name.firstName()} ${faker.name.lastName()}`,
+                totalIncomeYear: faker.random.number({ min: 1000, max: 1000000 }),
+                yourIdeaProject: faker.lorem.words(55),
+                mainContactEmail: faker.internet.email(),
+                mainContactPhone: "0345 4 10 20 30",
+                organisationType: faker.random.arrayElement(orgTypes),
+                projectDateRange: {
+                    endDate: toDateParts(moment().add(30, 'weeks')),
+                    startDate: toDateParts(moment().add(18, 'weeks'))
+                },
+                bankAccountNumber: faker.finance.account(),
+                projectTotalCosts: faker.random.number({ min: 301, max: 400 }),
+                seniorContactName: {
+                    lastName: faker.name.lastName(),
+                    firstName: faker.name.firstName()
+                },
+                seniorContactRole: "trustee",
```

Comment:
This will probably be invalid based on the `organisationType` above – have a look at the logic [here](https://github.com/biglotteryfund/blf-alpha/blob/master/controllers/apply/awards-for-all/lib/roles.js#L88).

One solution would be to hardcode the org type to something that matches this role (in this case, `UNINCORPORATED_REGISTERED_CHARITY`)

Reply:
Although not sure if this data really needs to be valid!